essential-tests/src/testFixtures/kotlin/org/stellar/anchor/platform/AbstractIntegrationTests.kt

```diff
@@ -20,16 +23,79 @@ import org.stellar.walletsdk.horizon.SigningKeyPair
 
 abstract class AbstractIntegrationTests(val config: TestConfig) {
   companion object {
-    const val TEST_MEMO = "22bf7341574e4b1082516a2e84a8"
-    const val TEST_PAYMENT_ID = "939223448317953"
-    const val TEST_PAYMENT_AMOUNT = "1.0000000"
-    const val TEST_STELLAR_TRANSACTION_HASH =
-      "1238508fd0e8f5a56507a51044aa741ea67b65f2d8044e281a212e580b8dc9d8"
-    const val TEST_STELLAR_TRANSACTION_SOURCE_ACCOUNT =
-      "GABCKCYPAGDDQMSCTMSBO7C2L34NU3XXCW7LR4VVSWCCXMAJY3B4YCZP"
-    const val TEST_STELLAR_TRANSACTION_DEST_ACCOUNT =
-      "GBDYDBJKQBJK4GY4V7FAONSFF2IBJSKNTBYJ65F5KCGBY2BIGPGGLJOH"
-    const val TEST_ASSET_USDC = "USDC:GBBD47IF6LWK7P7MDEVSCWR7DPUWV3NY3DTQEVFL4NAT4AQH3ZLLFLA5"
+    const val JSON_RPC_VERSION = "2.0"
+
+    // the search and replace values are injected into the expected values
+    const val TX_ID_KEY = "%TX_ID%"
+    const val RECEIVER_ID_KEY = "%RECEIVER_ID%"
+    const val SENDER_ID_KEY = "%SENDER_ID%"
+
+    // the test payment values
+    const val TEST_PAYMENT_MEMO = "22bf7341574e4b1082516a2e84a8"
+    const val TEST_PAYMENT_DEST_ACCOUNT = "GBDYDBJKQBJK4GY4V7FAONSFF2IBJSKNTBYJ65F5KCGBY2BIGPGGLJOH"
```

Comment:
Is this account only used for the Platform API tests?

Reply:
It is also used for SEP-10 account. I reuse the account because it has no other payment transactions. It would be more reliable to use it.